apps.go

```diff
@@ -0,0 +1,44 @@
+package slack
+
+import (
+	"context"
+	"net/url"
+)
+
+// AppsUninstallResponse contains our Auth response from the auth.revoke endpoint
+type AppsUninstallResponse struct {
```

Comment:
seems unnecessary..... only thing of value returned by these methods is an error?

Reply:
Or success . . . Not sure if you're proposing a change . . .

Reply:
I'm proposing your remote this type altogether and only return an error. this type doesn't provide any additional value to the public api.

Reply:
But, it's holding the ACTUAL response from slack. I modeled it off of other functions, but, obviously picked the wrong one.

Could you please point me to another slack endpoint that returns like you're suggesting? I'm happy to make any changes, but, I'm not sure exactly what you're suggesting . . . . particularly since almost ALL api endpoints return SlackResponse + any custom data, and this particular endpoint has no custom data to return.

Reply:
and the actual response provides no new information. other methods have actual data to return this one doesn't.

nothing in the SlackResponse contains any new or useful information that the error (or lack thereof) will contain.

>Could you please point me to another slack endpoint that returns like you're suggesting?

Yes actually I can, but I'm not terribly inclined to do so. why? because said functions existence is completely immaterial. following a pattern because its a pattern only makes sense when the conditions for using the pattern are the same.

your argument that every function does this is not convincing for why this particular function should do it. nor is your claim that every function does it correct.